internal/core/command/init.go

```diff
@@ -75,3 +85,21 @@ func (b *Bootstrap) BootstrapHandler(_ context.Context, _ *sync.WaitGroup, _ sta
 
 	return true
 }
+
+// loadV2Routes creates a new command-query router and handles the related mux.Router initialization for API V2 routes.
+func (b *Bootstrap) loadV2Routes(dic *di.Container, lc logger.LoggingClient) {
+	handlers := []delegate.Handler{}
+	if b.inDebugMode {
+		handlers = append(handlers, debugging.New(lc).Handler)
+	}
+
+	routing.Initialize(
+		dic,
+		b.muxRouter,
+		handlers,
+		common.V2Routes(
```

Comment:
Not clear how the Core Command specific routes are loaded. Might be helpful to explain in a comment in each of these loadV2Routes funcs.

Reply:
This will become self-evident once the follow-on service-specific endpoint example is delivered.  For the record, service-specific controllers are included in a slice (currently empty) passed to `common.V2Routes()` whose output is subsequently passed to `routing.Initialize()`.